src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketCapabilitiesClientImpl.java

```diff
@@ -26,12 +26,12 @@
      */
     public static final long CAPABILITIES_CACHE_DURATION =
             parsePositiveLongFromSystemProperty(CAPABILITIES_CACHE_DURATION_KEY, 360000);
+    private static Supplier<AtlassianServerCapabilities> capabilitiesCache;
```

Comment:
Any reason this moved from where it was before into the public variables section?

Reply:
Responding to a checkstyle warning, I think so static variables are grouped together